_shared_content/develop/guides/formats/best_practices/authentications.md

```diff
@@ -0,0 +1,520 @@
+{
+  "event": {
+    "category": ["authentication"],
+    "type": ["start"]
+  },
+  "user": {
+    "id": "1111111111"
+  },
+  "action": {
+    "outcome": "success"
+  },
+  "client": {
+    "ip": "1.2.3.4"
+  }
+}
+```
+
+#### Salesforce
+
+```JSON
+{
+  "EVENT_TYPE": "Login", # (1)
+  "USER": "john.doe@example.com", # (3)
+  "LOGIN_STATUS": "Success", # (2)
+  "IP_ADDRESS": "192.168.0.1",
+  "LOGIN_TIME": "2023-07-03T10:15:00Z",
+  "DEVICE_TYPE": "Desktop",
+  "BROWSER": "Chrome",
+  "PLATFORM": "Windows 10",
+  "LOGIN_GEO_LOCATION": "San Francisco, CA",
+  "SESSION_ID": "ABC123XYZ",
+  "LOGIN_URL": "https://login.salesforce.com"
+}
+```
+
+1. The EVENT_TYPE allows to determine the event.category and event.type
+2. The outcome of the event
+3. The email of the principal
+
+will result into this ECS document
+
+```json
+{
+  "event": {
+    "category": ["authentication"],
+    "type": ["start"]
+  },
+  "user": {
+    "email": "john.doe@example.org"
+  },
+  "action": {
+    "outcome": "success"
+  },
+  "client": {
+    "ip": "192.168.0.1"
+  },
+  "url": {
+    "original": "https://login.salesforce.com"
+  }
+}
+```
+
+#### RSA SecurID
+
+```
+11:23:02,069, example.intranet, audit.runtime.com.rsa.ims.authn.impl.AuthenticationBrokerImpl, INFO, da0011b4f66e4b7e86f90f9dd6e937e7,0e34d92f7c6549b19ed28471c02a049b,5.6.7.8,1.2.3.4,AUTHN_LOGIN_EVENT,13002,SUCCESS,AUTHN_METHOD_SUCCESS,e7ec7ff59d604a2ba3fa09067bbd65a4-L0+/miv3k62B,39b1319237f946428aecf267190b537d,09f1f5fc30e947ce9e564d5a91745091,000000000000000000001000e0011000,HDTCO04,HDTCO04,,559eb5ec2d43408cbce2a43b65eafe8c,000000000000000000001000e0011000,1.2.3.4,source.hostname,000000000000000000002000f1022000,SecurID_Native,,,AUTHN_LOGIN_EVENT,6,4,,,,,da624c0ecf554764953fcc346b999682,000523656192,,
+```
+
+1. The value AUTHN_LOGIN_EVENT allows to determine the event.category and event.type
+2. The value SUCCESS allows to define the outcome
+3. The 
+
+will result into this ECS document
+
+```json
+{
+  "event": {
+    "category": ["authentication"],
+    "type": ["start"]
+  },
+  "user": {
+    "id": "39b1319237f946428aecf267190b537d",
+    "name": "HDTCO04"
+  },
+  "action": {
+    "outcome": "success"
+  },
+  "client": {
+    "ip": "1.2.3.4"
+  },
+  "sekoiaio": {
+    "client": {
+      "name": "source.hostname"
+    },
+    "server": {
+      "name": "example.intranet"
+    }
+  }
+}
+```
+
+### How to write a good smart-description for authentication events
+
+From the previous samples, we can build the following smart-description:
+
+```json
+{
+  "value": "{user.name} sign in from {sekoiaio.client.name} ({client.ip})",
+  "conditions": [
+     {"field": "event.category", "value": "authentication"},
+     {"field": "event.type", "value": "start"},
+     {"field": "user.name"},
+     {"field": "sekoiaio.client.name"},
+     {"field": "client.ip"}
+  ]
+}
+```
+
+For the AWS cloudtrail event, this smart-description will result into:
+
+`1111111111 sign in from source.hostname (1.2.3.4)`
```

Comment:
```suggestion
`1111111111 signed in from source.hostname (1.2.3.4)`
```

Is `source.hostname` what you meant to write in this sentence of should it be replaced by the value of the field ?